Bottom: d5b095fa4d942f98444c9d42010387e8918e01e2
Top:    d8df1bf6e695131bcbfdff02157e35df2b668ec0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-01 16:04:55 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
